app/uk/gov/hmrc/timetopay/arrangement/model/model.scala

```diff
@@ -184,3 +184,112 @@ object modelFormat {
+      taxpayer             = ttpArrangement.taxpayer,
+      bankDetails          = ttpArrangement.bankDetails,
+      schedule             = ttpArrangement.schedule,
+      desArrangement       = ttpArrangement.desArrangement match {
+        case None => None
+        case Some(desSubmissionRequest: DesSubmissionRequest) =>
+          Some(desSubmissionRequest)
+      }
+    )
+  }
+}
+
+case class AnonymousTaxpayer(
+    selfAssessment: AnonymousSelfAssessment)
+
+object AnonymousTaxpayer {
+  implicit val format: OFormat[AnonymousTaxpayer] = Json.format
+
+  implicit def makeTaxpayerAnonymous(taxpayer: Taxpayer): AnonymousTaxpayer = {
+    AnonymousTaxpayer(selfAssessment = taxpayer.selfAssessment)
+  }
+}
+
+case class AnonymousSelfAssessment(
+    utr: String
+)
+
+object AnonymousSelfAssessment {
+  implicit val format: OFormat[AnonymousSelfAssessment] = Json.format
+
+  implicit def makeSelfAssessmentAnonymous(selfAssessment: SelfAssessment): AnonymousSelfAssessment = {
+    AnonymousSelfAssessment(utr = selfAssessment.utr)
+  }
+}
+
+case class AnonymousDesSubmissionRequest(ttpArrangement: DesTTPArrangement)
+
+object AnonymousDesSubmissionRequest {
+  implicit val format: OFormat[AnonymousDesSubmissionRequest] = Json.format
+
+  implicit def makeDesSubmissionRequestAnonymous(desSubmissionRequest: DesSubmissionRequest): AnonymousDesSubmissionRequest = {
+    AnonymousDesSubmissionRequest(ttpArrangement = desSubmissionRequest.ttpArrangement)
+  }
+}
+
+case class TTPArrangementResponse(
+    _id:                  String,
+    createdOn:            String,
+    paymentPlanReference: String,
+    directDebitReference: String,
+    taxpayer:             AnonymousTaxpayer,
+    bankDetails:          BankDetails,
+    schedule:             PaymentSchedule,
+    desArrangement:       Option[DesSubmissionRequestResponse]
+)
+
+object TTPArrangementResponse {
+  implicit val format: OFormat[TTPArrangementResponse] = Json.format[TTPArrangementResponse]
+
+  implicit def buildTTPArrangementResponse(anonymousTTPArrangement: AnonymousTTPArrangement): TTPArrangementResponse = {
+    TTPArrangementResponse(
+      _id                  = anonymousTTPArrangement._id,
+      createdOn            = anonymousTTPArrangement.createdOn.toString,
+      paymentPlanReference = anonymousTTPArrangement.paymentPlanReference,
+      directDebitReference = anonymousTTPArrangement.directDebitReference,
+      taxpayer             = anonymousTTPArrangement.taxpayer,
+      bankDetails          = anonymousTTPArrangement.bankDetails,
+      schedule             = anonymousTTPArrangement.schedule,
+      desArrangement       = anonymousTTPArrangement.desArrangement match {
+        case None => None
+        case Some(anonymousDesSubmissionRequest: AnonymousDesSubmissionRequest) =>
+          Some(
+            DesSubmissionRequestResponse(
+              ttpArrangement   = anonymousDesSubmissionRequest.ttpArrangement,
+              letterAndControl = None
```

Comment:
Currently this line does not achieve the `"letterAndControl" : {}` in the GET response as it as before migration.

I suggest that this change to the payload is not substantive.